README.md

```diff
@@ -78,13 +78,13 @@ This will set up a new application on your Heroku account using this repo as a t
 
 -->
 
-### Import to Replit
+### Import to CodeSandbox
 
-Create a Replit account at [Replit - Sign up](https://replit.com/signup), and then click on this button:
+Sign in to your CodeSandbox account at [CodeSandbox - Sign in](https://codesandbox.io/signin?utm_source=landingpage) and then click on this button:
```

Comment:
what if users don't have an account? before, with Replit, we had a step to sign up

Reply:
CodeSandbox doesn't have a "sign up" button, they call it "sign in" and there you can also create a new account

Reply:
the new version is here:

```
Sign in to your CodeSandbox account at [CodeSandbox - Sign in](https://codesandbox.io/signin?utm_source=landingpage).
```

1. you can drop the `utm_source`
2. on that page linked above, they mention "register", so let's use that word
   ![Screenshot 2024-05-06 at 17 21 21](https://github.com/upleveled/express-guest-list-api-memory-data-store/assets/1935696/83c32dc6-2de3-4cb7-8bbc-20858573e1e6)
3. "your CodeSandbox account" assumes that the user already has an account, which may be confusing (and also incorrect)

Reply:
Changed the beginning of the sentence to "Login or register", and dropped `utm_source` in the URL
[c5cd73d](https://github.com/upleveled/express-guest-list-api-memory-data-store/pull/294/commits/c5cd73d2dbfc7da1511ebbadb99a6a0aac2cbb56)

Reply:
current version is this, probably needs a grammar fix:
```
Login or register to [CodeSandbox - Sign in](https://codesandbox.io/signin?landingpage).
```

Reply:
Changed to "log in" 
[85003f4](https://github.com/upleveled/express-guest-list-api-memory-data-store/pull/294/commits/85003f41df9731c8b53e91e97389f179b0053420)

Reply:
that's a good improvement, but not what I meant

it's saying `log in or register "to"` - "to" is incorrect

Reply:
changed it to `on` [818baac](https://github.com/upleveled/express-guest-list-api-memory-data-store/pull/294/commits/818baace1a15198a62ed5169d5487bd99fb0d067)